src/functionalTest/resources/updatecase/valid-request.json

```diff
@@ -58,7 +59,7 @@
               "document_binary_url": "binary-url",
               "document_filename": "987654321-123456789.pdf"
             },
-            "control_number": "987654321",
```

Comment:
use different number than exception_record's doc

Reply:
This is an invalid value. Can you use a valid value?